cmd/gokr-packer/packer.go

```diff
@@ -404,6 +404,39 @@ func findDontStart() (map[string]bool, error) {
 	return contents, nil
 }
 
+func findWaitForClock() (map[string]bool, error) {
+	waitForClockPaths, err := findPackageFiles("waitforclock")
+	if err != nil {
+		return nil, err
+	}
+
+	if len(waitForClockPaths) == 0 {
+		return nil, nil // no waitforclock.txt files found
+	}
+
+	buildPackages := buildPackagesFromFlags()
+
+	contents := make(map[string]bool)
+	for _, p := range waitForClockPaths {
+		pkg := strings.TrimSuffix(strings.TrimPrefix(p.path, "waitforclock/"), "/waitforclock.txt")
+		if !buildPackages[pkg] {
+			log.Printf("WARNING: waitforclock.txt file %s does not match any specified package (%s)", pkg, flag.Args())
+			continue
+		}
+		packageConfigFiles[pkg] = append(packageConfigFiles[pkg], packageConfigFile{
+			kind:         "wait for the clock before start",
```

Comment:
How about: “wait for clock synchronization before start”?

Reply:
Sure, makes sense.